traffic_ops/testing/api_contract/v4/test_cdns.py

```diff
@@ -0,0 +1,68 @@
+"""Api Contract Test Case for cdns endpoint"""
+import json
+import logging
+import pytest
+
+# Create and configure logger
+logger = logging.getLogger()
+
+
+@pytest.fixture(name="get_cdn_data")
+def get_cdn_prereq_data():
+    """PyTest Fixture to store prereq data for cdns endpoint"""
+    # Response keys for cdns endpoint
+    with open('prerequisite_data.json', encoding="utf-8", mode='r') as prereq_file:
+        data = json.load(prereq_file)
+    cdn_data = data["cdns"]
+    return cdn_data
+
+
+def test_get_cdn(to_session, get_cdn_data, cdn_prereq):
+    """Test step to validate keys, values and data types from cdns endpoint response
```

Comment:
missing punctuation at the end of the sentence

Reply:
Added punctuation.